package/boot/uboot-envtools/files/ipq40xx

```diff
@@ -31,6 +31,7 @@ ubootenv_mtdinfo () {
 }
 
 case "$board" in
+8dev,jalapeno |\
```

Comment:
why do you need access to uboot env? I don't see anything changing something in the env. Furthermore, the uboot env write protected.

I'm quite sure it can be dropped.